src/app/shared/models/boltzModels.ts

```diff
@@ -1,3 +1,22 @@
+export interface BlockHeight {
+  key: string;
+  value: number;
+}
+
+export interface BoltzInfo {
+  version: string;
+  node: string;
+  network: string;
+  nodePubkey: string;
+  autoSwapStatus: string;
+  blockHeights: BlockHeight[];
```

Comment:
The actual response is a map:
```json
  "blockHeights": {
    "BTC": 171,
    "L-BTC": 150
  },
```
But im not sure why doc generator says its an array:
https://docs.boltz.exchange/v/boltz-client/grpc#getinforesponse

Reply:
Updated.